src/commands/cloud-login.ts

```diff
@@ -0,0 +1,198 @@
+import { input, select, confirm } from "@inquirer/prompts";
+import { Endpoint, ShellConfig } from "../lib/config";
+import { hostname } from "os";
+import { Command } from "@oclif/core";
+import { underline, blue } from "chalk";
+
+const DEFAULT_NAME = "cloud";
+const DB = process.env.FAUNA_URL ?? "https://db.fauna.com";
+const AUTH = process.env.FAUNA_AUTH ?? "https://auth.console.fauna.com";
+
+type Regions = {
+  [key: string]: Region;
+};
+
+class Region {
+  name: string;
+  secret: string;
+
+  constructor(name: string, secret: string) {
+    this.name = name;
+    this.secret = secret;
+  }
+
+  endpointName(base: string) {
+    if (this.name === "global") {
+      return base;
+    } else {
+      return `${base}-${this.name}`;
+    }
+  }
+}
+
+export default class CloudLoginCommand extends Command {
+  static description = "Prompt the user to log into a Fauna account.";
```

Comment:
Minor: I think the description should say to the user what they will accomplish with the command, so this should be phrased as "Log in to a Fauna account".

Reply:
I like it, I'll update